### PR TITLE
July24 updates

### PR DESCRIPTION
**Pull request by shuvashish76**
- New: [Apps] FadCam, Aaru, FDTracker, Element X, Mercurygram, FocusPodcast, Tack, Canta, Geto, Network Survey, Nix-on-Droid, Json List, KeyStoreViewer, Gadgetbridge, Launch Chat, Materials Live Wallpaper, Paperize, Peristyle, Slideshow Wallpaper, Wave Lines Live Wallpaper, Rain (#403) | [Games] Minute Maze, Chip Defense, Flowit

- Other minor updates: Openlib, Feeder, LogFox, Breezy Weather, aShell You (moved from Games/Miscellaneous to Apps/System), freeDictionaryApp,

### CODE REVIEW

**Comment by tjsuominen** [comment]
The Wallpaper list seems not yet updated, or does this work?

**Comment by shuvashish76** [comment]
> The Wallpaper list seems not yet updated, or does this work?

@tjsuominen I've added the recently _updated_ wallpaper apps. Which one doesn't work for you ?

**Comment by tjsuominen** [comment]
Can not find the listed wallpaper apps from the updated (?) list.

File: README.md [hunk 1]
```diff
@@ -189,6 +189,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 ### • Camera
 
 * [**Cryptocam**](https://gitlab.com/cryptocam/cryptocam) <sup>**[[F-Droid](https://f-droid.org/app/com.tnibler.cryptocam)]**</sup>
+* [**FadCam**](https://github.com/anonfaded/FadCam) <sup>**[[IzzyOnDroid](https://apt.izzysoft.de/fdroid/index/apk/com.fadcam)]**</sup>
```
Comment (offa) [inline README.md:192]: Wired, the project readme links to F-Droid, but there's no package, nor a rfp :see_no_evil:.

File: README.md [hunk 2]
```diff
@@ -333,6 +334,8 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 
 ### • Finance
 
+* [**Aaru**](https://github.com/KSSidll/Arru) <sup>**[[IzzyOnDroid](https://apt.izzysoft.de/fdroid/index/apk/com.kssidll.arru)]**</sup>
+* [**FDTracker**](https://github.com/Abhay-cloud/Fixed-Deposit-Tracker) <sup>**[[IzzyOnDroid](https://apt.izzysoft.de/fdroid/index/apk/dev.abhaycloud.fdtracker)]**</sup>
```
Comment (offa) [inline README.md:338]: The 'official' name seems to be *Fixed Deposit Tracker*.

```suggestion
* [**Fixed Deposit Tracker**](https://github.com/Abhay-cloud/Fixed-Deposit-Tracker) <sup>**[[IzzyOnDroid](https://apt.izzysoft.de/fdroid/index/apk/dev.abhaycloud.fdtracker)]**</sup>
```

File: README.md [hunk 3]
```diff
@@ -604,6 +611,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**AntennaPod**](https://github.com/AntennaPod/AntennaPod) <sup>**[[F-Droid](https://f-droid.org/app/de.danoeh.antennapod)]**</sup>
 * [**Audio Anchor**](https://github.com/flackbash/AudioAnchor) <sup>**[[F-Droid](https://f-droid.org/app/com.prangesoftwaresolutions.audioanchor)]**</sup>
 * [**Escapepod**](https://codeberg.org/y20k/escapepod) <sup>**[[F-Droid](https://f-droid.org/app/org.y20k.escapepod)]**</sup>
+* [**FocusPodcast**](https://github.com/allentown521/FocusPodcast) <sup>**[[F-Droid](https://f-droid.org/en/app/allen.town.focus.podcast)]**</sup>
```
Comment (offa) [inline README.md:614]: correction:
```suggestion
* [**FocusPodcast**](https://github.com/allentown521/FocusPodcast) <sup>**[[F-Droid](https://f-droid.org/app/allen.town.focus.podcast)]**</sup>
```

File: README.md [hunk 4]
```diff
@@ -764,8 +777,11 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**Extirpater**](https://gitlab.com/divested-mobile/extirpater) <sup>**[[F-Droid](https://www.f-droid.org/app/us.spotco.extirpater)]**</sup>
 * [**HexViewer**](https://github.com/Keidan/HexViewer) <sup>**[[F-Droid](https://f-droid.org/app/fr.ralala.hexviewer)]**</sup>
 * [**InviZible Pro**](https://github.com/Gedsh/InviZible) <sup>**[[F-Droid](https://f-droid.org/app/pan.alexander.tordnscrypt.stable)]**</sup>
+* [**Json List**](https://github.com/SlaVcE14/JsonList) <sup>**[[F-Droid](https://f-droid.org/app/com.sjapps.jsonlist)]**</sup>
 * [**Keep Alive**](https://github.com/keepalivedev/KeepAlive) <sup>**[[F-Droid](https://f-droid.org/app/io.keepalive.android)]**</sup>
 * [**Key Mapper**](https://github.com/keymapperorg/KeyMapper) <sup>**[[F-Droid](https://www.f-droid.org/app/io.github.sds100.keymapper)]**</sup>
+* [**KeyStoreViewer**](https://github.com/qdsfdhvh/KeyStoreViewer) <sup>**[[F-Droid](https://f-droid.org/app/com.seiko.keystoreviewer)]**</sup>
```
Comment (offa) [inline README.md:783]: This project seems very new (started a month ago with < 50 commits and only a single release). I'm not sure about this yet.

On the other hand, it's on F-Droid already and what might increase it's audience. So might be ok I think.

File: README.md [hunk 5]
```diff
@@ -898,6 +919,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**Block Puzzle Stone Wars**](https://github.com/SoltauFintel/blockpuzzle) <sup>**[[F-Droid](https://f-droid.org/app/de.mwvb.blockpuzzle)]**</sup>
 * [**LibreSudoku**](https://github.com/kaajjo/Libre-Sudoku) <sup>**[[F-Droid](https://www.f-droid.org/app/com.kaajjo.libresudoku)]**</sup>
 * [**Minesweeper - Antimine**](https://github.com/lucasnlm/antimine-android) <sup>**[[F-Droid](https://f-droid.org/app/dev.lucanlm.antimine)]**</sup>
+* [**Minute Maze**](https://gitlab.com/ygingras/minute-maze) <sup>**[[F-Droid](https://f-droid.org/app/net.ygingras.minutemaze)]**</sup>
```
Comment (offa) [inline README.md:922]: As KeyStoreViewer above, this is almost too new to make an assessment.

**Comment by offa** [comment]
Thanks, for your PR! :smile: 

It looks good so far, but as commented I'm not sure if *Minute Maze* and *KeyStoreViewer* fit here (yet).

**Comment by shuvashish76** [comment]
> I'm not sure if Minute Maze and KeyStoreViewer fit here (yet).

We don't have any such criteria. IMO we should promote new projects here...plus those are on official F-Droid anyway.
I've added my suggestion in #148 so that we can add various new projects & remove if their _last commits_ are too old i.e. probably project abandoned.

File: README.md (hunk 2 above)
Comment (shuvashish76) [reply README.md:338]: https://github.com/Abhay-cloud/Fixed-Deposit-Tracker#home-screen
They use short name "FDTracker" when installed.

Comment (offa) [reply README.md:338]: I see.A little confusing, but hat's just how it is. :smile:

**Comment by offa** [comment]
>  IMO we should promote new projects here...plus those are on official F-Droid anyway.

Yes, you are completely right :+1:.

**Review by offa — APPROVED** [review]

**Comment by anonfaded** [comment]
Thanks for featuring my app, FadCam :)

**Pull request merged**